ArbOwner.sol

```diff
@@ -142,6 +142,14 @@ interface ArbOwner {
         uint256 pricePerUnit
     ) external;
 
+    /// @notice Get how much L1 charges per non-zero byte of calldata
```

Comment:
I don't think you need the getter to be in both the ArbOwner.sol and ArbOwnerPublic.sol.
Let's just keep the one in the public contract.